integrations/enterprise-search.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Integration: Enterprise Search.
+ *
+ * @package Automattic\VIP\Integrations
+ */
+
+namespace Automattic\VIP\Integrations;
+
+/**
+ * Loads Enterprise Search VIP Integration.
+ */
+class EnterpriseSearchIntegration extends Integration {
+	/**
+	 * Returns `true` if Enterprise Search is already available e.g. customer code. We will use
+	 * this function to prevent loading of integration again from platform side.
+	 */
+	public function is_loaded(): bool {
+		return class_exists( '\Automattic\VIP\Search\Search' );
```

Comment:
Would something like:
```suggestion
		return class_exists( \Automattic\VIP\Search\Search::class );
```
...work? IDEs can usually help autocomplete and better syntax highlight this than a regular string that can have typos.